### DISCUSSION TITLE
problem with BrowserAnimationsModule in angular v17

### DISCUSSION THREAD

**Discussion by thiagomotaPE** [primefaces/community#1440] [PrimeNG] [answered]
when I import the BrowserAnimationsModule or the BrowserModule in the main module or in any other module 
the following error appears: 

![Captura de tela 2024-03-14 151043](https://github.com/primefaces/community/assets/87782152/c7452918-380c-4365-9876-91f67c933919)

obs: the BrowserModule is not imported into any other part of the application

**Comment by cetincakiroglu** [comment, marked as answer]
Unrelated with PrimeNG, remove if you have browser animation module imported. You don't need to import it from the top-level module anymore, it's now defined in your projects app.config.ts file. Just create a blank angular example in stackblitz and follow the structure. You can also visit: https://angular.dev/

**Reply by SaiPrakashSarisa** [reply]
I am stuck with the same error but, I haven't used browser animations in any import, still I am getting the same error.

**Reply by dalvarez-opt** [reply]
Have you imported CommonModule in your standalone component? or in the module where you are declaring your component?

**Reply by SaiPrakashSarisa** [reply]
Yes I have imported the CommonModule in my standalone component. Any way, my error got resolved by adding `provideAnimations()` method in the providers array in my **app.config.js** file.

`export const appConfig: ApplicationConfig = {
  providers: [
    provideAnimations(),
    provideRouter(routes),  
    provideHttpClient(withFetch(),withInterceptors([AuthInterceptor])),
  ]
};`

**Comment by sherrera-sw** [comment]
I fixed by adding this on app.config.ts:

```
import { ApplicationConfig, provideZoneChangeDetection } from '@angular/core';
import { provideRouter } from '@angular/router';
import { provideAnimations } from '@angular/platform-browser/animations';

import { routes } from './app.routes';

export const appConfig: ApplicationConfig = {
  providers: [
    provideZoneChangeDetection({ eventCoalescing: true }),
    provideRouter(routes),
    provideAnimations(),
  ],
};

```

![image](https://github.com/user-attachments/assets/d43da4eb-b0dc-48b9-9ba7-37db12ac9623)

Please update the get started! I lost too much time on this
